### PR TITLE
Update version info shown in doc for building LS plugins

### PR DESCRIPTION
**Pull request by dedemorton**
This fixes examples in issue #6124 

Also tweaked the language in the introduction so it doesn't seem like 2.0 is the new release.

### CODE REVIEW

File: docs/static/include/pluginbody.asciidoc [hunk 1]
```diff
@@ -852,7 +852,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "{plugintype}" }
 
   # Gem dependencies
-  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency 'logstash-core', '>= 5.0.0', '< 6.0.0'
```
Comment (suyograo) [inline docs/static/include/pluginbody.asciidoc]: This should be replaced by line:
`s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"`

File: docs/static/include/pluginbody.asciidoc [hunk 2]
```diff
@@ -910,12 +910,12 @@ Logstash plugins:
 [subs="attributes"]
 ----------------------------------
   # Gem dependencies
-  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency 'logstash-core', '>= 5.0.0', '< 6.0.0'
```
Comment (suyograo) [inline docs/static/include/pluginbody.asciidoc]: This should be replaced by line:
`s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"`

File: docs/static/include/pluginbody.asciidoc [hunk 3]
```diff
@@ -910,12 +910,12 @@ Logstash plugins:
...
   s.add_development_dependency 'logstash-devutils'
 ----------------------------------
 This gemspec has a runtime dependency on the core Logstash gem and requires that
-it have a version number greater than or equal to version 1.4.0 `'>= 1.4.0'`,
-and less than version 2.0 `'< 2.0.0'`.
+it have a version number greater than or equal to version 5.0.0 `'>= 5.0.0'`,
```
Comment (suyograo) [inline docs/static/include/pluginbody.asciidoc]: This should be dependency on logstash-core-plugin-api with constraints: ">= 1.60", "<= 2.99"

**Comment by dedemorton** [comment]
@suyograo I've made the changes that you requested. BTW, should the following statement say "logstash-core-plugin-api" instead of "logstash core gem"?

 "All plugins have a runtime dependency on the logstash core gem, and a development dependency on logstash-devutils. "

**Comment by suyograo** [comment]
@dedemorton correct. It should be:

> "All plugins have a runtime dependency on the logstash-core-plugin-api gem, and a development dependency on logstash-devutils. "

Once you make the change, please merge.

**Pull request closed without merging**